uwrt_rqt_plugins/.vscode/settings.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
This shouldnt be committed

Reply:
there's also a .vscode in the meta package, can I just add it to the gitignore?

Reply:
Yes

Reply:
issue fixed